prov/efa/src/rdm/rxr_pkt_type_base.c

```diff
@@ -122,24 +123,35 @@ int rxr_pkt_init_data_from_op_entry(struct rxr_ep *ep,
 
 	rxr_locate_iov_pos(op_entry->iov, op_entry->iov_count, tx_data_offset,
 			   &tx_iov_index, &tx_iov_offset);
-	desc = op_entry->desc[0];
+	iov_mr = op_entry->desc[tx_iov_index];
```

Comment:
Shouldn't this line be after L127?

Correct me if I'm wrong - first we should `assert(tx_iov_index < op_entry->iov_count);` before `op_entry->desc[tx_iov_index];`? Is it possible to segfault because of out of bound index?

Reply:
added assertion in new revision.